### DISCUSSION TITLE
You guys should add an IDE

### DISCUSSION THREAD

**Discussion by electric-otter** [ghostty-org/ghostty#4117] [Feature Requests, Ideas] [answered]
Developers code, and it seems like a good mix to add a tty and a IDE altogether! :smile: 
Similar discussions:
Macos opening ghostty with open -n command opens it in a new process

**Comment by acsetter** [comment]
You should check out https://neovim.io/

**Comment by mitchellh** [comment, marked as answer]
Not planned.
